tests/scenarios/compat-addon-classic-features-test.ts

```diff
@@ -131,3 +131,43 @@ appScenarios
       });
     });
   });
+
+appScenarios
+  .map('compat-addon-classic-features-virtual-scripts', () => {})
+  .forEachScenario(scenario => {
+    let app: PreparedApp;
+
+    Qmodule(`${scenario.name} - build mode`, function (hooks) {
+      hooks.before(async assert => {
+        app = await scenario.prepare();
+        let result = await app.execute('pnpm build');
+        assert.equal(result.exitCode, 0, result.output);
+      });
+
+      test('vendor.js script is emitted in the build', async function (assert) {
+        assert.true(lstatSync(`${app.dir}/dist/@embroider/core/vendor.js`).isFile());
+      });
+    });
+
+    Qmodule(`${scenario.name} - dev mode`, function (hooks) {
+      hooks.before(async () => {
+        app = await scenario.prepare();
+      });
+
+      test('vendor.js script is served', async function (assert) {
+        const server = CommandWatcher.launch('vite', ['--clearScreen', 'false'], { cwd: app.dir });
+        try {
+          const [, url] = await server.waitFor(/Local:\s+(https?:\/\/.*)\//g);
```

Comment:
```suggestion
          const [, url] = await server.waitFor(/Local:\s+(https?:\/\/.*)\//g, 180 * 1000);
```

Reply:
Thanks, I'll skip the test as discussed during the office hours, but that was probably the issue 👍